extforms/views.py

```diff
@@ -0,0 +1,285 @@
+from django.conf import settings
+from django.core.urlresolvers import reverse, reverse_lazy
+from django.shortcuts import render
+from django.template.loader import get_template
+from django.views.generic import TemplateView
+
+from workshops.forms import (
+	SWCEventRequestForm,
```

Comment:
Good catch! I was in the middle of configuring my editor so I ended up with mixed tabs and spaces.